### PR TITLE
ref(dynamic-sampling): Add Error Panel

### PR DESCRIPTION
**Pull request by priscilawebdev**
Added Error Panel + adjusts. It looks like below:

image

### CODE REVIEW

File: src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/filtersAndSampling.tsx [hunk 1]
```diff
@@ -47,36 +53,91 @@ class FiltersAndSampling extends AsyncView<Props, State> {
...
+          operator: 'and',
+          inner: [
+            {
+              operator: 'strEqualNoCase',
+              name: 'environments',
+              value: ['dev', 'prod'],
+            },
+            {
+              operator: 'strEqualNoCase',
+              name: 'userSegements',
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/filtersAndSampling.tsx:81]:
```suggestion
              name: 'userSegments',
```

File: src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/filtersAndSampling.tsx [hunk 2]
```diff
@@ -117,10 +189,24 @@ class FiltersAndSampling extends AsyncView<Props, State> {
...
+        <RulesPanel
+          rules={errorRules}
+          platformDocLink={platformDocLink}
+          onAddRule={this.handleAddErrorRule(platformDocLink)}
+          onEditRule={this.handleEditRule}
+          onDeleteRule={this.handleDeleteRule}
+        />
+        <TextBlock>
+          {t(
+            'The transaction order is limited. Traces must occur first and individual transactions must occur last. Any individual transaction rules before a trace rule will be disregarded. '
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/filtersAndSampling.tsx:201]: Are we going to give the user feedback that their rules are invalid when they do this?

File: src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/filtersAndSampling.tsx (hunk 1 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/filtersAndSampling.tsx]: I will delete it soon...it's just a mock :)

File: src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/filtersAndSampling.tsx (hunk 2 above)
Comment (priscilawebdev) [reply src/sentry/static/sentry/app/views/settings/project/filtersAndSampling/filtersAndSampling.tsx:201]: yes, I will implement this functionality together with drag-and-drop

**Review by markstory — APPROVED** [review]

**Pull request merged**